src/project-css/app/Console/Commands/enableCollection.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use App\Helpers\CollectionHelper;
+
+class enableCollection extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'collection:enable {collectioname}';
+
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Enable a Collection';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
```

Comment:
No return defined, what is doing the return or is this a procedural method that just enacts other methods? If so it should be @abstract

Reply:
All the commands just execute commands inside the handle() generally you will only get 2 types of returns from the handle() either a  

return $this->error('some error occurred');
return $this->info('command ran correctly'); 

either is shown to the user in the terminal.